apps/token/src/routes/governance/components/propose/proposal-form-vote-and-enactment-deadline.tsx

```diff
@@ -108,6 +126,9 @@ const ValidationForm = ({
 };
 
 interface EnactmentFormProps {
+  onEnactMinMax:
```

Comment:
chore (blocking): optional butI think this is always passed in? Or am I missing something

Reply:
No worries, yeah it needs to either be optional here (which actually makes less sense) or become part of the render logic for if the enactmentForm is rendered (I'll do that instead)

Reply:
Scratch that - they don't always get passed in